docs/asyncapi.yaml

```diff
@@ -0,0 +1,205 @@
+asyncapi: 2.4.0
```

Comment:
`, optional` for keys optional in message

Reply:
The async api spec only has a required flag for object schemas, https://www.asyncapi.com/docs/reference/specification/v2.0.0#schemaObject

Am I missing an optional?